hardware/hardware.mk

```diff
@@ -128,7 +128,7 @@ endef
 # data. Each rom entry is a 32-character string with 1/0 values. Each
 # line is a separate entry. This target generates the verilog for the rom.
 $(HARDWARE_PATH)/%.v: $(HARDWARE_PATH)/%.rom
-	python $(BASEJUMP_STL_DIR)/bsg_mem/bsg_ascii_to_rom.py $< \
+	/usr/bin/python $(BASEJUMP_STL_DIR)/bsg_mem/bsg_ascii_to_rom.py $< \
                bsg_bladerunner_configuration > $@
```

Comment:
Why is it necessary to use /usr/bin? That's a bit too OS-specific for my liking

Reply:
basejump scripts tend to use system python (we can say python2). So, using system python felt right. Agree, but it's what the python file intends to do: https://github.com/bespoke-silicon-group/basejump_stl/blob/0e3313335586c399c839249364fc209a6b45cb74/bsg_mem/bsg_ascii_to_rom.py#L1.

Reply:
Oh, and the actual reason I had to change is, when I'm in python3 environment, basejump_stl's python2 scripts fail.

Reply:
A better solution might be `env python2` or `env python3`. That is path independent 

See: https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash

Reply:
That means we would replace `python` with `python2` as the executable name? 

Another idea to treat the script as an executable and and let it handle which interpreter it invokes. Does this sound good?

Reply:
Sounds good to me, as long as the script doesn't hard code to use `/bin/env`

Reply:
Actually, it does. And, I think it's okay because the script is written for that interpreter? I understand the executable, rarely, might not be there, but not on any linux. 

But since python2 is officially deprecated, we might not find python2 executable in modern python distributions. So, explicitly searching for python2 binary using `env python2` is also a great idea.

@proftaylor, a note about python2 use in basejump_stl.